### PR TITLE
 #21 printf format

### PR DESCRIPTION
**Pull request by sec-matsunaga**
Correct the indication by the static analysis tool.
 Specify printf format explicitly.

### CODE REVIEW

File: voxel_grid/src/voxel_grid.cpp [hunk 1]
```diff
@@ -202,7 +202,7 @@ namespace voxel_grid {
       printf("Layer z = %u:\n",z);
       for(unsigned int y = 0; y < size_y_; y++){
         for(unsigned int x = 0 ; x < size_x_; x++){
-          printf((getVoxel(x, y, z)) == voxel_grid::MARKED? "#" : " ");
+          printf("%s", (getVoxel(x, y, z) == voxel_grid::MARKED)? "#" : " ");
```
Comment (r-kurose) [inline voxel_grid/src/voxel_grid.cpp]: You should use "%c".

Comment (sec-matsunaga) [reply voxel_grid/src/voxel_grid.cpp]: Certainly it is a char type.

Comment (sec-matsunaga) [reply voxel_grid/src/voxel_grid.cpp]: warning: format ‘%c’ expects argument of type ‘int’, but argument 2 has type ‘const char*’ [-Wformat=]

"#" : " " => '#' : ' '
change

Comment (r-kurose) [reply voxel_grid/src/voxel_grid.cpp]: OK.

**Pull request merged**